src/features/timeformat.js

```diff
@@ -1,11 +1,42 @@
 import moment from '../lib/moment.js';
 import { keyToCss } from '../utils/css_map.js';
+import { buildStyle } from '../utils/interface.js';
 import { pageModifications } from '../utils/mutations.js';
 import { getPreferences } from '../utils/preferences.js';
 
 let format;
 let displayRelative;
 
+export const styleElement = buildStyle(`
+[data-formatted-time] {
+  font-size: 0px !important;
+}
+
+[data-formatted-time]::before {
+  content: attr(data-formatted-time);
+  font-size: .78125rem;
+}
+
+[data-formatted-relative-time]::after {
+  content: attr(data-formatted-relative-time);
+  font-size: .78125rem;
+  display: inline-block;
+}
+
+[data-formatted-time][title]::before,
+[data-formatted-time][title]::after {
+  cursor: help;
+}
+
+${keyToCss('blogLinkWrapper')}:has(+ [data-formatted-time]) {
+  flex: none;
+}
+${keyToCss('blogLinkWrapper')} + [data-formatted-time] {
```

Comment:
the stylelint in my brain was activated here

```suggestion

${keyToCss('blogLinkWrapper')} + [data-formatted-time] {
```